plenum/server/node.py

```diff
@@ -538,6 +538,8 @@ def on_view_change_start(self):
         Notifies node about the fact that view changed to let it
         prepare for election
         """
+        self.view_changer._start_view_change_ts = self.utc_epoch()
```

Comment:
Can we also track the view change we start a chain of view changes in addition to the VC time?

Reply:
Do you mean, that it contains a list of (propogated_view_no, vc_start_time) tuple?

Reply:
I mean a new field (or a new item in a tuple). The new field should show the viewNo we start a view change from. We may have multiple rounds of viewNo changing during the view change.
Example: startViewNo=13, currentViewNo=14, viewChangeInProgress=True
so, it means that we were on viewNo=12, tried viewNo=13, failed for some reasons (a new primary is unavailable for some reasons for example as we use round robin) , start viewNo=14, and currently trying to complete viewNo=14